dotnet-desktop-guide/net/winforms/whats-new/net60.md

```diff
@@ -0,0 +1,202 @@
+
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventHandler?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.MinimumSplashScreenDisplayTime?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.MinimumSplashScreenDisplayTime?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.Font?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.Font?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.HighDpiMode?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.HighDpiMode?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.ApplyApplicationDefaults?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.HighDpiMode?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.HighDpiMode?displayProperty=fullName>
+
+## Updated APIs
+
+- <xref:System.Windows.Forms.Control.Invoke?displayProperty=fullName> now accepts <xref:System.Action?displayProperty=fullName> and <xref:System.Func%601?displayProperty=fullName> as input parameters.
+- <xref:System.Windows.Forms.Control.BeingInvoke?displayProperty=fullName> now accepts <xref:System.Action?displayProperty=fullName> as an input parameter.
+- <xref:System.Windows.Forms.DialogResult?displayProperty=fullName> is extended with the following members:
+
+  - `TryAgain`
+  - `Continue`
+
+- <xref:System.Windows.Forms.Form?displayProperty=fullName> is extended with the following property:
+
+  - <xref:System.Windows.Forms.Form.MdiChildrenMinimizedAnchorBottom?displayProperty=fullName>
+
+- <xref:System.Windows.Forms.MessageBoxButtons?displayProperty=fullName> is extended with the following member:
+
+  - `CancelTryContinue`
+
+- <xref:System.Windows.Forms.MessageBoxDefaultButton?displayProperty=fullName> is extended with the following member:
+
+  - `Button4`
+
+- <xref:System.Windows.Forms.LinkClickedEventArgs?displayProperty=fullName> has now a new constructor and extended with the following properties:
+
+  - <xref:System.Windows.Forms.LinkClickedEventArgs.LinkLength?displayProperty=fullName>
+  - <xref:System.Windows.Forms.LinkClickedEventArgs.LinkStart?displayProperty=fullName>
+
+- <xref:System.Windows.Forms.NotifyIcon.Text?displayProperty=fullName> is now limited to 127 characters (from 63).
+
+## Enhanced features
+
+- High DPI improvements.
+TODO: @dreddy-work to provide text.
```

Comment:
The text is here: https://github.com/dotnet/docs-desktop/pull/1199#discussion_r744406765